dspace-server-webapp/src/test/java/org/dspace/app/rest/WorkflowDefinitionRestRepositoryIT.java

```diff
@@ -239,7 +240,7 @@ public void getWorkflowDefinitionByCollectionId_nonValidUUID() throws Exception
         //When we call this facets endpoint
         getClient(token).perform(get(WORKFLOW_DEFINITIONS_ENDPOINT + "/search/findByCollection?uuid=" + nonValidUUID))
             //We expect a 400 Illegal Argument Exception cannot convert UUID
-            .andExpect(status().is(400))
+            .andExpect(status().is(HttpStatus.SC_BAD_REQUEST))
```

Comment:
@MarieVerdonck : This could be simplified by writing
`.andExpect(status().isBadRequest());`   You'll see similar code in other ITs in our codebase.